examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -0,0 +1,252 @@
+import {
+  addressToBytes,
+  getProgram,
+  intToBigEndian,
+  SignType,
+  stringToBytes,
+  TransactionType
+} from '@algorand-builder/algob';
+import { Runtime, StoreAccountImpl } from '@algorand-builder/algorand-js/build/src/index';
+import { assert } from 'chai';
+
+import { getAcc } from './common';
+
+const initialDonorBalance = 60000000;
+const initialCreatorBalance = 10000;
+const goal = 7000000;
+
+describe('Crowdfunding Tests', function () {
+  let creatorAccount = new StoreAccountImpl(initialCreatorBalance);
+  let escrowAccount = new StoreAccountImpl(0);
+  let donorAccount = new StoreAccountImpl(initialDonorBalance);
+
+  let runtime;
+  let program;
+  let flags;
+  let applicationId;
+  this.beforeAll(async function () {
+    runtime = new Runtime([creatorAccount, escrowAccount, donorAccount]);
+    program = getProgram('crowdFundApproval.teal');
+
+    flags = {
+      sender: creatorAccount.account,
```

Comment:
Should we rename `creatorAccount` to `creator` (similarly for others)? `creator.account` makes more sense than `creatorAccount.account`

Reply:
yes